scripts/exglobal_aero_analysis_finalize.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# exgdas_global_aero_analysis_finalize.py
+# exglobal_aero_analysis_finalize.py
```

Comment:
Good catch.  Same typo is in `exglobal_atm_analysis*` and `exglobal_atmens_analysis*` python scripts.

Reply:
@RussTreadon-NOAA Do you want me to correct the typo in this PR? Thanks.

Reply:
@RussTreadon-NOAA Done the corrections of these typos. Thanks.